From bb145c103c03eabf7adf79d486c7143ebed3ad78 Mon Sep 17 00:00:00 2001
From: Al Bailey <Al.Bailey@windriver.com>
Date: Tue, 10 Oct 2017 09:41:17 -0500
Subject: [PATCH] WRS: Patch1005:
 1005-Set-strict_variables-and-basemodulepath-in-puppet.co.patch

Also sets ordering = title-hash so that puppet 4 will run the same as puppet 3
---
 conf/puppet.conf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/conf/puppet.conf b/conf/puppet.conf
index bf48823..15674d7 100644
--- a/conf/puppet.conf
+++ b/conf/puppet.conf
@@ -4,3 +4,9 @@
 # - https://docs.puppetlabs.com/puppet/latest/reference/config_about_settings.html
 # - https://docs.puppetlabs.com/puppet/latest/reference/config_file_main.html
 # - https://docs.puppetlabs.com/puppet/latest/reference/configuration.html
+[main]
+    # Prevent the use of undefined variables
+    strict_variables = true
+    ordering = title-hash
+    # Set the path to TIS puppet modules
+    basemodulepath = /usr/share/puppet/modules:/usr/share/openstack-puppet/modules
-- 
1.8.3.1

